presto-main/src/main/java/io/prestosql/operator/NestedLoopBuildOperator.java

```diff
@@ -33,22 +35,24 @@
         private final int operatorId;
         private final PlanNodeId planNodeId;
         private final JoinBridgeManager<NestedLoopJoinBridge> nestedLoopJoinBridgeManager;
+        private final List<Integer> buildChannels;
```

Comment:
Why aren't build channels extracted by pruning projection? It seems like it should be the case. If it isn't, we already lost because build data was transferred over the network.